src/components/App/App.tsx

```diff
@@ -1,11 +1,13 @@
 import { useState } from "react"
 import { Button } from "../Button"
 import { Modal } from "../Modal"
+import { Tab, Tabs } from "../Tabs"
 import { TextInput } from "../TextInput"
 import "./App.styles.css"
 
 export function App() {
   const [isOpen, setIsOpen] = useState<boolean>(false)
+  const [currentTab, setCurrentTab] = useState(0)
```

Comment:
+1 to using IDs instead of indexes. Indexes work fine, but if you're generating dynamic tabs out of some data ingestion, indexes could end up not being reliable

Reply:
I haven't converted this over to IDs just yet, but that's the next step 👍